docs/ecctl-release-notes.asciidoc

```diff
@@ -26,3 +27,4 @@ include::release_notes/v1.0.0.adoc[]
 include::release_notes/v1.0.0-beta3.adoc[]
 include::release_notes/v1.0.0-beta2.adoc[]
 include::release_notes/v1.0.0-beta1.adoc[]
+include::release_notes/v1.7.0.adoc[]
```

Comment:
This guy should be reordered at the top of the `include::` list, since otherwise it'll show up at the bottom and it's not what we want.